### DISCUSSION TITLE
help upgrading v1 usage to v4

### DISCUSSION THREAD

**Discussion by CollinMonahanLab49** [panva/jose#294] [Q&A]
How to rewrite the following against version 4 of this library?

```js
jose.JWK.asKey(buffer, {
    alg: 'A256GCM',
    use: 'enc',
    kid: 'mock-encryption-key',
  })
```

Thank you

**Comment by panva** [comment]
you can just use the buffer to encrypt/decrypt

**Reply by CollinMonahanLab49** [reply]
Trying to call SignJWT.  Trying this led to `Key must be of type CryptoKey. Received an instance of Buffer`.

**Reply by panva** [reply]
Please post the full snippet. These tiny splinters don't make overall sense. You've used to import a buffer as an encryption key that you passed to `jose.JWT.sign`? I'm not making heads or tails of this.
